doc/smart-switch/pmon/smartswitch-pmon.md

```diff
@@ -676,26 +678,10 @@ fantray0    N/A  fantray0.fan      55%       intake     Present        OK  20230
 fantray1    N/A  fantray1.fan      56%       intake     Present        OK  20230728 06:41:17
 ```
 
-#### 3.4.1 Reboot Cause
+#### 3.4.1 Reboot Cause CLIs
```

Comment:
@rameshraghupathy Please specify which entity or service will update the chassisStateDB

Reply:
The DPU PMON will be responsible to update the the reboot-cause onto the chassisStateDB.
@prgeor

Reply:
@rameshraghupathy understood its PMON. Which agent/daemon inside pmon?

Reply:
@prgeor  "Though how DPU pmon updates this is vendor dependent, it is recommended to use the sonic telemetry agent to align with the existing SONiC implementation."

Reply:
@rameshraghupathy can you update this section "3.4.1 Reboot Cause CLIs"  according to new reboot low?

Reply:
@prgeor Updated section 3.4.1  according to the new reboot work-flow